lib/beaker-rspec/spec_helper.rb

```diff
@@ -32,6 +32,8 @@
     debug: ENV['BEAKER_DEBUG'] || ENV['BEAKER_debug'] || ENV.fetch('RS_DEBUG', nil),
     destroy: ENV['BEAKER_DESTROY'] || ENV['BEAKER_destroy'] || ENV.fetch('RS_DESTROY', nil),
     optionsfile: ENV['BEAKER_OPTIONS_FILE'] || ENV['BEAKER_options_file'] || ENV.fetch('RS_OPTIONS_FILE', nil),
+    vagrant_memsize: ENV['BEAKER_VAGRANT_MEMSIZE'] || ENV.fetch('BEAKER_vagrant_memsize', nil),
+    vagrant_cpus: ENV['BEAKER_VAGRANT_CPUS'] || ENV.fetch('BEAKER_vagrant_cpus', nil),
```

Comment:
I'd lean to only uppercase since nothing is using lowercase now. If people don't like the mixed use of upper and lower case they can convert to all uppercase
```suggestion
    vagrant_memsize: ENV['BEAKER_VAGRANT_MEMSIZE'],
    vagrant_cpus: ENV['BEAKER_VAGRANT_CPUS'],
```